sourcegraph/sourcegraph.proto

```diff
@@ -809,8 +809,9 @@ service Changesets {
 	rpc Update(ChangesetUpdateOp) returns (ChangesetEvent);
 
 	// Merge merges the head branch of a changeset into its base branch and
-	// pushes the resulting merged base.
-	rpc Merge(ChangesetMergeOp) returns (pbtypes.Void);
+	// pushes the resulting merged base. It returns the resulting update event.
+	// If no merge occurred, it returns nil.
```

Comment:
I believe so, for reference the same thing happens in [Changeset.Update](https://github.com/sourcegraph/go-sourcegraph/blob/changeset-merge-event/sourcegraph/sourcegraph.proto#L807-L809)